### PR TITLE
Various ghost role QoL; moving holding facility; removing energy net

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

Mildly pissed contractors used the holding facility without telling me, so I moved it to a much smaller area, without ghost dojo peeps there. Made ghost dojo able to use the standard "ghost" verb with zero penalty, instead of having an action. Ninja no longer have energy nets; just removed.

## Why It's Good For The Game

Stuff is all a bit half-finished. Energy nets are the most vile, execrable gay baby jailing the game has to offer, completely removing people from a round with no chance of being saved in any respect, explicitly penalizing them for ghosting. It's really, really bad, and I've removed it. The new holding facility prevents any possible metagaming from contractor kidnapees from meeting ghost dojo people.

## Changelog
:cl:
add: A new, much more barebones holding facility for contractors.
del: Energy nets are gone.
tweak: Various ghost roles are now easier to ghost as.
tweak: The candles in the dojo no longer make the place hotter and hotter over time.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -297,12 +297,17 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
...
+	if(sig_flags & COMPONENT_BLOCK_GHOSTING)
 		return
 
+	if(sig_flags & COMPONENT_DO_NOT_PENALIZE_GHOSTING)
+		penalty = 0
+
 	if(stat != DEAD)
 		succumb()
-	if(stat == DEAD)
+	if(stat == DEAD || sig_flags & COMPONENT_FREE_GHOSTING)
```
Comment (Ghommie) [inline code/modules/mob/dead/observer/observer.dm:311]: Copy these signal changes to `mob/camera/verb/ghost()`, and also apply these new signals to `mob/proc/ghostize` too.

**Review by Ghommie — CHANGES_REQUESTED** [review]
You also forgot to remove the ninja net objective.

**Comment by Putnam3145** [comment]
I didn't remove it but I didn't see it get added anywhere, so I figured it was already "removed".

**Comment by Ludox235** [comment]
THANK GOD NOW WE DON'T HAVE ANY MORE NINJAS BE LIKE

*click button
*you are now gay baby jailed in a barebone place for the rest of the ground

File: code/modules/mob/dead/observer/observer.dm (hunk 1 above)
Comment (Putnam3145) [reply code/modules/mob/dead/observer/observer.dm:311]: Done.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**